### PR TITLE
Expose collection download count in the api

### PR DESCRIPTION
**Pull request by bmclaughlin**
https://issues.redhat.com/browse/AAH-2236

[noissue]

### CODE REVIEW

**Review by newswangerd — COMMENTED** [review]
I think the code here looks fine. @awcrosby is this the how you want this feature to work?

File: pulp_ansible/tests/functional/api/collection/v3/test_collection_version_download_count.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    ).download_count
+
+
+def test_cv_download_count(
+    galaxy_v3_content_collections_index_versions_download_count_api_client,
+    install_scenario_distribution,
+    pulp_admin_user,
+    ansible_dir_factory,
+):
+    pass
```
Comment (gerrod3) [inline pulp_ansible/tests/functional/api/collection/v3/test_collection_version_download_count.py:56]: Is this passing over this test?

File: pulp_ansible/tests/functional/api/collection/v3/test_collection_version_download_count.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
+from os import path
+import subprocess
+import pytest
+
+from pulpcore.client.pulp_ansible import AnsibleRepositorySyncURL
+from pulp_smash.pulp3.bindings import monitor_task
```
Comment (gerrod3) [inline pulp_ansible/tests/functional/api/collection/v3/test_collection_version_download_count.py:6]: `monitor_task` is a now pytest fixture available from pulpcore.
```suggestion
```

File: pulp_ansible/tests/functional/api/collection/v3/test_collection_version_download_count.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+def test_cv_download_count(
+    galaxy_v3_content_collections_index_versions_download_count_api_client,
+    install_scenario_distribution,
+    pulp_admin_user,
+    ansible_dir_factory,
+):
+    pass
+    collection_name = ANSIBLE_DEMO_COLLECTION
+    namespace, name = ANSIBLE_DEMO_COLLECTION.split(".")
```
Comment (gerrod3) [inline pulp_ansible/tests/functional/api/collection/v3/test_collection_version_download_count.py:58]: It doesn't seem that you are consistent with using `collection_name` after assigning to it.

File: pulp_ansible/tests/functional/api/collection/v3/test_collection_version_download_count.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    return dl_count_api_client.read(
+        namespace=namespace,
+        name=name,
+        distro_base_path=distro_base_path,
+        path=distro_base_path,
+        version=version,
+    ).download_count
+
+
+def test_cv_download_count(
```
Comment (gerrod3) [inline pulp_ansible/tests/functional/api/collection/v3/test_collection_version_download_count.py:50]: I think this test needs a check to see if the download log is enabled (it is a setting we can check, correct?)

**Review by gerrod3 — CHANGES_REQUESTED** [review]

File: pulp_ansible/tests/functional/api/collection/v3/test_collection_version_download_count.py (hunk 1 above)
Comment (bmclaughlin) [reply pulp_ansible/tests/functional/api/collection/v3/test_collection_version_download_count.py:56]: Seems likely.  I removed it locally and it still passed.  Too many ctrl-z's when fixing the formatting earlier.  :smile:

**Comment by bmclaughlin** [comment]
@gerrod3 Thank you for the review!  Unfortunately I'm going to close this PR in favor of a different approach that should be more efficient.  I'll reach out when that PR is up and ready for review.

**Pull request closed without merging**